mattermost.tera

```diff
@@ -0,0 +1,34 @@
+---
+whiskers:
+  version: 2.1.0
```

Comment:
I believe this only works on `2.2.0` as the colours aren't in the root context for `2.1.0`
```suggestion
  version: 2.2.0
```

Reply:
You would be correct! An old copy paste was where that come from.